api/project.go

```diff
@@ -252,13 +252,20 @@ func ValidateProjectDBNameTemplate(template string) error {
 
 // FormatTemplate formats the template.
 func FormatTemplate(template string, tokens map[string]string) (string, error) {
-	keys := getTemplateTokens(template)
+	keys, fixed := getTemplateTokens(template)
 	for _, key := range keys {
 		if _, ok := tokens[key]; !ok {
 			return "", fmt.Errorf("token %q not found", key)
 		}
 		template = strings.ReplaceAll(template, key, tokens[key])
 	}
+	for _, key := range fixed {
+		quoteKey := regexp.QuoteMeta(key)
+		if quoteKey != key {
+			template = strings.Replace(template, key, quoteKey, 1)
+		}
+
+	}
 	return template, nil
```

Comment:
How about simply
```go
return regexp.QuoteMeta(template), nil
```

Reply:
This cannot be done, only fixed string can be escaped.

Reply:
This seems to be the only way to do a single escape for a specific fixed string, and if the template is escaped, the regular expression is not what is expected